.travis.yml

```diff
@@ -1,37 +1,39 @@
 language: ruby
-rvm:
-  - 1.8.7
-  - 2.0.0
 env:
-  - ENV=nox
-  - ENV=python
-  - ENV=python3
-  - ENV=ruby
+  global:
+    - DEPS=$HOME/deps
+    - PATH=$DEPS/bin:$PATH
 matrix:
-  exclude:
-    - rvm: 2.0.0
   include:
-    - rvm: 2.0.0
-      env: ENV=ruby
+    - env: ENV=nox
+      rvm: 1.8.7
+      addons: { apt: { packages: [vim-nox] } }
+    - env: ENV=python
+      rvm: 1.8.7
+      addons: { apt: { packages: [python2.7-dev] } }
+    - env: ENV=python3
+      rvm: 1.8.7
+      addons: { apt: { packages: [python3-dev] } }
+    - env: ENV=ruby
+      rvm: 1.8.7
+    - env: ENV=ruby
+      rvm: 2.0.0
 install: |
   git config --global user.email "you@example.com"
   git config --global user.name "Your Name"
-  sudo apt-get update -y
 
   if [ "$ENV" == "nox" ]; then
-    sudo apt-get install -y vim-nox
-    sudo ln -s /usr/bin/vim /usr/local/bin/vim
+    mkdir -p ${DEPS}/bin
+    ln -s /usr/bin/vim.nox ${DEPS}/bin/vim
     return
   fi
 
-  C_OPTS="--with-features=huge --disable-gui "
+  C_OPTS="--prefix=$DEPS --with-features=huge --disable-gui "
   case "$ENV" in
     python)
-      PACKS=python2.7-dev
       C_OPtS+=--enable-pythoninterp
       ;;
     python3)
-      PACKS=python3-dev
       C_OPtS+=--enable-python3interp
```

Comment:
Hmm, I missed this the last time, but the lowercase `t` here is not intentional, right?